llvm/include/llvm/BinaryFormat/Wasm.h

```diff
@@ -229,6 +229,7 @@ enum WasmSegmentFlag : unsigned {
   WASM_SEG_FLAG_STRINGS = 0x1,
   WASM_SEG_FLAG_TLS = 0x2,
   WASM_SEG_FLAG_RETAIN = 0x4,
```

Comment:
Can we not just reuse the `WASM_SEG_FLAG_RETAIN` for these segments?

How does the ELF linker prevent GC of these segments?